pkg/kv/kvclient/rangefeed/config.go

```diff
@@ -155,6 +157,16 @@ func WithFiltering(withFiltering bool) Option {
 	})
 }
 
+// WithInvoker makes an option to invoke the rangefeed tasks such as running the
+// the client and processing events emitted by the client with a caller-supplied
+// function, which can make it easier to introspect into work done by a given
+// caller as the stacks for these tasks will now include the caller's invoker.
```

Comment:
Maybe mention that the error is unused currently just to avoid confusion down the line.

Reply:
what do you mean by "the error is unused" ?